taxi/models.py

```diff
@@ -0,0 +1,47 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=63)
+    country = models.CharField(max_length=63)
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["name"],
+                name="unique_manufacturer"
+            )
+        ]
+
+    def __str__(self) -> str:
+        return f"Name: {self.name} Country: {self.country}"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=63)
+    manufacturer = models.ForeignKey(
+        to=Manufacturer,
+        on_delete=models.DO_NOTHING,
```

Comment:
why did u choose do nothing option ?

Reply:
because even if we manufacturer stops existing, cars will still be there